src/sentry/static/sentry/app/views/settings/organizationRelay/list/code.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+
+type Props = {
+  text: string;
+};
+
+// TODO(Priscila): Make this component reusable and available in the storybook
+const Code = ({text}: Props) => <Wrapper>{text}</Wrapper>;
```

Comment:
Why not use children for the contents?

Reply:
yes I use the children here, but I was thinking of accepting only the string type